analysis_options.yaml

```diff
@@ -8,7 +8,7 @@ include: package:lints/recommended.yaml
 analyzer:
   errors:
     unused_element: error
-    unused_import: error
+    unused_import: ignore
```

Comment:
Let's not change this. If an import is not used, let's remove it.

Reply:
fixed